examples/rn-connection-and-error/frontend/app/atlas-app-services/config.ts

```diff
@@ -16,7 +16,8 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+import Config from 'react-native-config';
 import {BSON} from 'realm';
 
-export const ATLAS_APP_ID = 'YOUR_APP_ID';
+export const ATLAS_APP_ID = Config.ATLAS_APP_ID || 'YOUR_APP_ID';
```

Comment:
Shouldn't we throw here with a more meaningful error, guiding the user to create a .env file?